polkadot-parachains/encointer-runtime/src/lib.rs

```diff
@@ -523,6 +528,8 @@ impl pallet_collective::Config<CouncilCollective> for Runtime {
 	type DefaultVote = pallet_collective::PrimeDefaultVote;
 	type MaxMembers = CouncilMaxMembers;
 	type WeightInfo = weights::pallet_collective::WeightInfo<Runtime>;
+	type SetMembersOrigin = MoreThanHalfCouncil;
+	type MaxProposalWeight = ();
```

Comment:
@OverOrion this will brick our council and needs fixing. See https://github.com/integritee-network/parachain/pull/217